CharacterCreation/01_01_Races_Human

```diff
@@ -0,0 +1,145 @@
+developed and well-used. 
+Terran Common, Eastern Common, Westlich, and Mandar are the most common human languages.
+
+[Humans gain 1 additional language and pick up new languages at twice the normal rate]
+
+
+===== Names ===== 
+
+Humans can be named in as many ways as there are Terran languages. Usually, these names 
+have some significant meaning to the parents, but that is not always the case. There is no 
+standard of naming that is generally agreed upon.
+
+== Human Name Examples ==
+
+* Matthew Kesler
+* Lupita Ugara
+* Tomoyo Usagi
+
+===== Available Traits =====
+
+Pitching Arm:
+All humans have the inherent bone and muscle structure necessary for extremely 
+accurate throwing. Not all humans practice this technique, but you are one who 
+has mastered this seemingly simple aspect of your anatomy.
+[Negates the aiming action necessary to throw accurately]
+
+Stubborn: 
+All Humans have a hardiness to them that cannot always be explained by logic.
+Some humans are just plain hard to kill, for no other apparent reason than they 
+just refuse to die. 
+[Roll Advantage on Death Saving Throws]
+
+
+==============================
+Human Sub-Species
+==============================
+
+===== Terrans =====
+
+Size: Medium
+Home System: Keplar
+
+Appearance: 
+Heavily resembling the historic humans, there is little variation from even 20th 
+century humans in this pool. The Terrans left Sol behind in the 24th Century, 
+forming a new colony in the Kepler system. 
+
+
+== Available Traits ==
+
+Very Normal: 
+Blending in has always come early to you. People tend to trust you. You seem to 
+belong no matter where you find yourself. 
+[Advantage on Persuade & Deceive ]
+
+Sixth Sense:
+Colloquially often known as the ‘gut’ instinct, sixth sense is that feeling that 
+many Terrans get when they feel something is off. There are different levels of 
+Sixth Sense, and many function in different ways. Some Terrans get goosebumps, 
+while others experience a hair-raising feeling; it can be very hard to get anything 
+past Humans with this sense tuned. 
+[Grants you an extra sense. DM’s discretion for application]
+
+
+
+===== Eastern Earthlings =====
+
+Size: Small/Medium
+Home System: Sol
+
+Inhabiting the areas of Earth east of equator, these humans have adapted to the humid,
+swampy heat of their half of the planet. Small and lean, these humans rarely reach over 
+5’ in height, and are much lighter than their Terran counterparts. As the new earth 
+conditions have stabilized, these humans have been beginning to rebuild some sense of 
+agriculture, but only recently have they ceased their nomadic patterns.
+
+== Available Traits ==
+
+Endurance: 
+Generations of Nomadic behavior have made moving long distances easy for those of 
+Eastern Earthling descent. 
+[First 2 Actions Sprinting are free of penalty. 1 Recovery Action required
+for every 2 Sprinting Actions]
```

Comment:
Sprinting actions have been modified in C2.0, penalties don't stack indefinitely like they used to:
https://github.com/bleehu/Compound_X/blob/37f9e8d2537dc1844a7e710c06b2375f2572d812/01_01_Basic_Rules.txt#L859
So this reading of Endurance will be kind of break-even in terms of penalties. The longer you sprint, the worse it looks compare to C2.0 sprinting

Reply:
Do you have a suggestion for a boon that will work better with C2.0?